### PR TITLE
Week 3 - Project business site by Jenny

### PR DESCRIPTION
**Pull request by jenquach**

### CODE REVIEW

File: code/style.css [hunk 1]
```diff
@@ -1,7 +1,383 @@
...
+  
+  .hero h2 {
+    margin-top: 0;
+  }
+
+  .wrapper{
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    background-color: transparent;
+  }
```
Comment (nehrwein) [inline code/style.css:317]: What does background-color do here?

File: code/style.css [hunk 2]
```diff
@@ -1,7 +1,383 @@
...
+    .hero h1 {
+      margin-bottom: 0;
+      text-align: center;
+    }
+    
+    .hero h2 {
+      margin-top: 0;
+    }
+  }
+}
```
Comment (nehrwein) [inline code/style.css:383]: Your media queries for tablet and desktop are almost identical. I suggest, you get rid of all code that's double to make it easier to maintain and read 🙂

**Review by nehrwein — CHANGES_REQUESTED** [review]
General Requirements
1.	Code follows Technigo’s code guidelines:
•	you are using comments
•	your code follows semanthic HTML

2.	Your site is published on Netlify.

Blue Level Requirements
🔵 Blue Level (Minimum Requirements)
•	Style your page  -ok!
•	Site should have a header with responsive image/video -ok!
•	Site should have a signup form with at least three different input types -ok!
•	Site should have an FAQ accordion, built with JavaScript -ok!

GOOD JOB! :)

I would like to hear from you about my feedback on your code.
If you have any questions contact me! :)

File: code/style.css [hunk 3]
```diff
@@ -1,7 +1,383 @@
...
+
+@media screen and (min-width: 901px) {
+  .nav-toggle-label {
+    display: none;
+  }
+
+  header {
+    display: grid;
+    grid-template-columns: .5fr auto minmax(600px, 1fr);
+  }
```
[the remark this answers is not in the document]
Comment (jenquach) [reply code/style.css:265]: nothing, it's removed.

File: code/index.html [hunk 4]
```diff
@@ -1,30 +1,109 @@
...
-    <section class="accordion">
-      <!-- Accordion -->
+          <label> 
+            <input class="radio" type="radio" name="info"> Intermediate Level
+          </label>
+
+          <label> 
+            <input class="radio" type="radio" name="info"> Advanced Level
+          </label>
+        </div>
```
[the remark this answers is not in the document]
Comment (jenquach) [reply code/index.html:70]: Thank you for this, I love how detailed you are. It's fixed now. :)

File: code/style.css (hunk 2 above)
Comment (jenquach) [reply code/style.css:383]: Yes, I didn't have time to have a look at it in the end. I was suppose to fix it today.

File: code/style.css [hunk 5]
```diff
@@ -1,7 +1,383 @@
...
+.hero::before {
+  content: "";
+  position: absolute;
+  background: url(lotus.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center center;
+  filter: brightness(60%);
+}
+
```
[the remark this answers is not in the document]
Comment (jenquach) [reply code/style.css:123]: I tried all kind of things when I wanted to get my footer to work and then in the end I forgot to remove this part....

File: code/style.css [hunk 6]
```diff
@@ -1,7 +1,383 @@
...
+.nav-toggle-label::before {
+  bottom: 30px;
+}
+
+.nav-toggle-label::after {
+  top: 30px;
+}
+
```
[the remark this answers is not in the document]
Comment (jenquach) [reply code/style.css:98]: haha, yes.. there is a 45 min youtube video I followed if you are interested. :)

**Comment by jenquach** [comment]
Thank you for the code review Birgit! I really like how detailed you are, especially with the httpbin. I have now updated my code and commited it.

**Pull request closed without merging**